plugins/Widgetize/vue/src/WidgetPreview/WidgetPreviewIframe.vue

```diff
@@ -6,14 +6,14 @@
 
 <template>
   <div id="embedThisWidgetIframe">
-    <label for="embedThisWidgetIframeInput">&rsaquo; Embed Iframe</label>
+    <label for="embedThisWidgetIframeInput" v-html="translate('Widgetize_EmbedIframe')"></label>
```

Comment:
When using `v-html` we always add a `$sanitize` to ensure no bad html can be inserted.
```suggestion
    <label for="embedThisWidgetIframeInput" v-html="$sanitize(translate('Widgetize_EmbedIframe'))"></label>
```